psychopy/experiment/components/apriltag/__init__.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Part of the PsychoPy library
+# Copyright (C) 2002-2018 Jonathan Peirce (C) 2019-2021 Open Science Tools Ltd.
+# Distributed under the terms of the GNU General Public License (GPL).
+
+from __future__ import absolute_import, print_function
+
+from contextlib import contextmanager
+from os import path
+from pathlib import Path
+from psychopy.experiment.components import image
+from psychopy.experiment.components.image import ImageComponent, Param, getInitVals
+from psychopy.localization import _translate
+
+
+class AprilTagComponent(ImageComponent):
+    """An event class for presenting image-based AprilTag stimuli"""
+
+    categories = ['Stimuli']
+    targets = ['PsychoPy', 'PsychoJS']
+    iconFile = Path(__file__).parent / 'apriltag.png'
+    tooltip = _translate('AprilTag: present AprilTag image')
+
+    _apriltag_images_dir = Path(__file__).parent / 'apriltag_images'
```

Comment:
It would be better if this was visually distinct from Image components - maybe a Pupil Core logo to the bottom right of the image symbol? Similar to how the Eyetracking components each have a different icon next to the eye. Happy to send over the raw vector files for any icons you need, or I could whip something up for you if you'd prefer.